### PR TITLE
#120: Distiguish between fast and slow tests

### LINKED ISSUE #120: Distiguish between fast and slow tests
## Situation

The ETC has a couple of long-running integration tests. While they are necessary, they are to expensive to always run and should be used as regression tests mainly.

## Acceptance Criteria

1. Expensive tests marked
1. POM contains two different profiles for tests: fast and slow
1. By default only fast tests run

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #120.

### CODE REVIEW

File: doc/changes/changes_3.4.1.md [hunk 1]
```diff
@@ -6,10 +6,27 @@ Code name: Workaround for log rotation bug in docker-db
...
 
+Since this change required longer running integration tests, we added three build profiles to the Maven configuration:
+
+* fast
+* slow
+* expensive
+
+"Fast" means all test that don't take longer than a second or two. "Slow" test can run for a handful of minutes and "expensive" tests run longer than 10 minutes.
+
+The default is now "fast" for regular builds and "slow" for Travis CI builds.
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_3.4.1.md:17]: What is a regular build?

File: pom.xml [hunk 2]
```diff
@@ -66,6 +66,29 @@
...
+    <profile>
+      <id>slow</id>
+      <properties>
+        <test.excludeTags>expensive</test.excludeTags>
+      </properties>
+    </profile>
+    <profile>
+      <id>expensive</id>
+      <properties>
+        <test.excludeTags></test.excludeTags>
```
Comment (AnastasiiaSergienko) [inline pom.xml:88]: Do you need this if it's empty?

**Review by AnastasiiaSergienko — APPROVED** [review]
Approved with minor questions

File: pom.xml (hunk 2 above)
Comment (redcatbear) [reply pom.xml:88]: Maybe not, but I find it more expressive this way.

File: doc/changes/changes_3.4.1.md (hunk 1 above)
Comment (redcatbear) [reply doc/changes/changes_3.4.1.md:17]: Any build you start that is not a CI build.

**Pull request merged**